WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -132,6 +132,7 @@
 
     private OnMagicLinkRequestInteraction mListener;
     private String mToken = "";
+    private boolean mInhibitMagicLogin;
```

Comment:
> Negative booleans are confusing.

Ah, I see your point @kwonye ! Indeed, it would require an inverse of logic and even though I'm not too familiar with the login related code, it seems it wouldn't be that simple. It would require to search for all the places that magic-link login is supposed to be enabled and guard them with the flag. For example, the helper functions in `ActivityLauncher` will need to pass the new flag as an extra set to true while now they just don't assuming the magic-login is the default mode anyway.

Would it be OK for you if we left it as it is with this PR? It seems the inversion would slightly be out-of-scope of this PR. Let me know what you think! Thanks!